src/scenes/game.js

```diff
@@ -1,17 +1,85 @@
+	    this.leftFrameArray = data.playerLeftFrames;
+	    this.rightFrameArray = data.playerRightFrames;
+	    //This is the array representing the stats of a player. Strength is [0] and speed is [1]
+		this.leftStrength = data.playerLeftStats[0];
+		this.leftSpeed = data.playerLeftStats[1];
+		this.rightStrength = data.playerRightStats[0];
+		this.rightSpeed= data.playerRightStats[1];
+	}
+	
 	preload() {
-	  	
+
 	}
 
 	create() {
-	 	
+		//THIS WHOLE THING IS TESTING
+		console.log(this.choiceLeft);
+		console.log(this.leftStrength);
+		console.log(this.leftSpeed);
+
+		//Because we get a spritesheet object from the character select screen, we need to create the sprite based off of that object.
+		let jousterLeft = this.physics.add.sprite(this.choiceLeft.x, this.choiceLeft.y, this.choiceLeft.texture.key, this.choiceLeft.frame.name);
+		jousterLeft.flipX = true;
+		let jousterRight = this.physics.add.sprite(this.choiceRight.x, this.choiceRight.y, this.choiceRight.texture.key, this.choiceRight.frame.name);
+
+		//This is an attempt to create a unified animation set so we don't need spaghetti code
+		//Basically, we get information from the choice of character, and modify the spritesheet and frames for each type of animation
+		this.anims.create({
+		key: 'runningLeft',
+		//repeat -1 means it loops
+			 repeat: -1,
+			 frameRate: 10,
+			 //Basically you get the key of the certain texture, and using the array passed from the character selection, determine the beginning and end frames
+			 frames: this.anims.generateFrameNames(this.choiceLeft.texture.key, {start: this.leftFrameArray[0], end: this.leftFrameArray[1]})
+		});
+
+		this.anims.create({
+		key: 'runningRight',
+		//repeat -1 means it loops
+			 repeat: -1,
+			 frameRate: 10,
+			 //Basically you get the key of the certain texture, and using the array passed from the character selection, determine the beginning and end frames
+			 frames: this.anims.generateFrameNames(this.choiceRight.texture.key, {start: this.rightFrameArray[0], end: this.rightFrameArray[1]})
+		});
+
+		this.anims.create({
+		key: 'attackingLeft',
+		//repeat -1 means it loops
+			 repeat: -1,
+			 frameRate: 10,
+			 //Basically you get the key of the certain texture, and using the array passed from the character selection, determine the beginning and end frames
+			 frames: this.anims.generateFrameNames(this.choiceLeft.texture.key, {start: this.leftFrameArray[2], end: this.leftFrameArray[3]})
+		});
+
+		this.anims.create({
+		key: 'attackingRight',
+		//repeat -1 means it loops
+			 repeat: -1,
+			 frameRate: 10,
+			 //Basically you get the key of the certain texture, and using the array passed from the character selection, determine the beginning and end frames
+			 frames: this.anims.generateFrameNames(this.choiceRight.texture.key, {start: this.rightFrameArray[2], end: this.rightFrameArray[3]})
+		});
+		//End of animations here
+
```

Comment:
40 to 77 is the new animations, feel free to remove the old ones. Instead of having a bunch of separate animations for each character, I just pass in the specific frame information to these functions. Now we just need 2 sets of animations, one for the left and right characters.